### PR TITLE
Hybridize

### PR DESCRIPTION
**Pull request by dlockhart**
I'm not going to register this with webcomponentsjs.org, since it's really for our own internal use and doesn't need to be publicly discoverable.

### CODE REVIEW

File: d2l-ajax-framed.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <link rel="import" href="../iron-ajax/iron-ajax.html">
 
 <dom-module id="d2l-ajax-framed">
-	<template>
+	<template strip-whitespace>
```
Comment (ryantmer) [inline d2l-ajax-framed.html]: Huh, didn't know about `strip-whitespace`. Neat!

File: package.json [hunk 2]
```diff
@@ -4,27 +4,25 @@
...
-    "postinstall": "bower install",
-    "serve": "polyserve --cors",
-    "test": "npm run test:lint:js && npm run test:lint:wc && npm run test:unit",
-    "test:lint:js": "eslint --ext .js,.html . test/ demo/",
-    "test:lint:wc": "polymer lint --rules=polymer-2-hybrid",
-    "test:unit": "polymer test --skip-plugin sauce"
+    "postinstall": "polymer install --variants",
+    "test": "npm run lint && npm run test:unit:local",
+    "lint:js": "eslint *.js *.html test/*.html test/*.js demo/*.html",
+    "lint:wc": "polymer lint",
```
Comment (ryantmer) [inline package.json]: I was under the impression this defaults to `src/` without any params, is that no longer the case? (I was explicitly calling out globs in polymer.json - unless that's no longer necessary?)

File: wct.conf.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
 				{
 					"browserName": "safari",
 					"platform": "OS X 10.12",
-					"version": ""
```
Comment (ryantmer) [inline wct.conf.json]: iirc the empty string gives you "latest", do we want 10.0 specifically for some reason?

Comment (dlockhart) [inline wct.conf.json]: Mostly I was just making it consistent with the other components. I don't mind being explicit though seeing as Safari makes major version bumps so rarely -- I think of it more like IE at this point.

File: package.json (hunk 2 above)
Comment (dlockhart) [inline package.json]: With the new CLI it became much more clever, as long as your project adheres to their common setup.
https://www.polymer-project.org/2.0/docs/tools/polymer-cli-commands#lint

File: d2l-ajax-framed.html (hunk 1 above)
Comment (dlockhart) [inline d2l-ajax-framed.html]: Yeah it's awesome... removes so many weird spacing issues.

File: package.json (hunk 2 above)
Comment (ryantmer) [inline package.json]: Huh - yeah, projects not following the same structure as theirs I was having trouble with, but :man_shrugging: If it works, it works! :)

**Pull request merged**